debian/control

```diff
@@ -0,0 +1,12 @@
+Source: restic
+Section: admin
+Priority: optional
+Maintainer: Vladimir Vinogradenko <vladimirv@ixsystems.com>
+Build-Depends: debhelper-compat (= 12), wget
+Standards-Version: 4.4.0
+Homepage: https://github.com/restic/restic/releases
+
+Package: restic
+Architecture: amd64
+Description: restic is a backup program that is fast, efficient and secure
```

Comment:
Duplicate sentences in the `Description:`

Reply:
These are different descriptions (short and long description), the build will fail if I only leave the first line.